src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/MSBuildRestoreUtility.cs

```diff
@@ -891,6 +895,23 @@ private static RestoreLockProperties GetRestoreLockProperites(IMSBuildItem specI
                 IsPropertyTrue(specItem, "RestoreLockedMode"));
         }
 
+        private static RestoreAuditProperties GetRestoreAuditProperties(IMSBuildItem specItem)
+        {
+            string enableAudit = specItem.GetProperty("NuGetAudit");
+            string auditLevel = specItem.GetProperty("NuGetAuditLevel");
```

Comment:
Can we use the constants in this context?

```suggestion
            string enableAudit = specItem.GetProperty(ProjectBuildProperties.NuGetAudit);
            string auditLevel = specItem.GetProperty(ProjectBuildProperties.NuGetAuditLevel);
```

Reply:
Unfortunately no. `ProjectBuildProperties` is defined in NuGet.PackageManagement, which has a ProjectReference to NuGet.Commands. Therefore, we cannot add a circular reference from NuGet.Commands back to NuGet.PackageManagement in order to use the API.

Why the const strings are defined in NuGet.PackageManagement instead of NuGet.Commands, I don't know. But I'm not a "reduce duplication at all costs!!!!!" purist. Once we ship a version of NuGet that uses "NuGetAudit" and "NuGetAuditLevel" MSBuild properties, it becomes part of our public API, and therefore effectively set in stone permanently. The benefit to reducing duplication is when a value changes, you don't have to change multiple places. But this value will never change, so there's effectively little benefit.